lib/ts/recipe/emailpassword/api/implementation.ts

```diff
@@ -2,13 +2,15 @@ import { APIInterface, APIOptions } from "../";
 import { logDebugMessage } from "../../../logger";
 import Session from "../../session";
 import { SessionContainerInterface } from "../../session/types";
-import { GeneralErrorResponse, User } from "../../../types";
+import { GeneralErrorResponse, User, UserContext } from "../../../types";
 import { listUsersByAccountInfo, getUser } from "../../../";
 import AccountLinking from "../../accountlinking/recipe";
 import EmailVerification from "../../emailverification/recipe";
 import { RecipeLevelUser } from "../../accountlinking/types";
 import RecipeUserId from "../../../recipeUserId";
 import { getPasswordResetLink } from "../utils";
+import MultiFactorAuthRecipe from "../../multifactorauth/recipe";
+import { MFAFlowErrors } from "../../multifactorauth/types";
 
 export default function getAPIImplementation(): APIInterface {
```

Comment:
a todo left for this is that @porcellus will get back to us about which security checks need to be done for mfa based on https://supertokens.com/docs/thirdpartyemailpassword/common-customizations/account-linking/security-considerations